tests/Tests/Random/Array.elm

```diff
@@ -0,0 +1,58 @@
+module Tests.Random.Array exposing (suite)
+
+import Array
+import Expect
+import Fuzz
+import Random
+import Random.Array
+import Set
+import Test exposing (Test)
+
+
+suite : Test
+suite =
+    Test.describe "shuffle"
+        [ Test.test "shuffle 10 elements with given seedRoot" <|
+            \() ->
+                let
+                    initialList =
+                        List.range 0 9
+
+                    ( shuffledArray, _ ) =
+                        Random.step (Random.Array.shuffle (Array.fromList initialList)) (Random.initialSeed 0)
+                in
+                shuffledArray
+                    |> Array.toList
+                    |> Expect.equalLists [ 4, 8, 5, 1, 0, 9, 7, 6, 2, 3 ]
+
+        --
+        , Test.fuzzWith { runs = 1 } Fuzz.int "test existing all of the 10k elements" <|
```

Comment:
So I get why we're using `runs = 1` with a 10k item array. But could we say, abstract the bodies of each test into functions that take the array size as an argument, and then run a 10 or 100 item test the normal 100 times, and also the 10k test once?

Reply:
Sure we could. I just would like keep one test case with 10k for "critical test" so the rest of them I'll make as you proposed.

Reply:
Yes, definitely keep the 10k case.